code/game/objects/effects/decals/crayon.dm

```diff
@@ -666,29 +735,129 @@ obj/item/scroll/attackby(obj/item/I, mob/living/carbon/human/M)
 		qdel(frodo)
 	return
 
-//ritual knife spell procs
+// Scribe: Gives you the Scribe perk. This is a requirement for inscribing scrolls with the spells below.
+/obj/effect/decal/cleanable/crayon/proc/scribe_spell(mob/living/carbon/human/M)
+	var/datum/reagent/organic/blood/B = M.get_blood()
+	var/mob/living/carbon/human/H = M
+	for(var/obj/item/paper/P in oview(1)) // Requires a paper
```

Comment:
stack 5 papers, use it, get -125 hp

Reply:
Shame on them for being dumb enough to do this! Just use one paper!!
Same argument could be said about not erasing blood writings on the floor, use book on rune get -400 blood from 6 spells that weren't erased >:|